src/osp/UserInputHandler.h

```diff
@@ -298,6 +298,10 @@ class UserInputHandler
     void config_register_control(std::string const& name, bool holdable,
             std::initializer_list<ButtonVarConfig> vars);
 
+    void config_register_control(std::string const&& name,
+        bool holdable,
```

Comment:
But remember, you need const& and &&, for the name parameter, as two seperate functions.

Reply:
Are they not?

Reply:
Currently you have one function that takes const&&_ which isn't really a thing